packages/Molecules/Card/CardBox.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="rpl-card-box">
+    <div class="rpl-card-box-items">
+      <div class="rpl-card-box-item" v-for="(item, index) in cards" :key="index">
+        <component :is="item.name" v-bind="item.data" :trimFieldEventBus="isTrimmed(item.name) ? eventBus : null" :trimFieldUpdateOnResize="false"></component>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import RplCardEmergencyContact from './CardEmergencyContact.vue'
+
+export default {
+  name: 'RplCardBox',
+  props: {
+    cards: Array
+  },
+  components: {
+    RplCardEmergencyContact
+  },
+  data () {
+    return {
+      eventBus: new Vue(),
+      // Define which cards use cardtrimfield. These will be given the eventBus
+      // object and their resize will be invoked on carousel size changes.
+      trimmedCards: [
+        'rpl-card-emergency-contact'
+      ]
+    }
+  },
+  methods: {
+    isTrimmed (name) {
+      return (this.trimmedCards.indexOf(name) >= 0)
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/style";
+
+$rpl-card-box-padding: ($rpl-space * 15) 0 ($rpl-space * 9) !default;
+$rpl-card-box-margin: 0 (-$rpl-space-3) !default;
+$rpl-card-box-background-color: rgba(rpl-color('primary'), .15) !default;
```

Comment:
I will suggest to make the Ripple components to fit the alpha theme, so maybe use our `light neutral` color as background here, and replace it with other colour in vicpol project.